Makefile

```diff
@@ -390,6 +390,16 @@ aws-machine-controller-image: build/aws-machine-controller/Dockerfile $(BINDIR)/
 	docker tag $(AWS_MACHINE_CONTROLLER_IMAGE) $(AWS_MACHINE_CONTROLLER_MUTABLE_IMAGE)
 	docker tag $(AWS_MACHINE_CONTROLLER_IMAGE) $(AWS_MACHINE_CONTROLLER_PUBLIC_IMAGE)
 
+# Push our Docker Images to the integrated registry:
+integrated-registry-push:
+	# WARNING: this will fail if logged in as system:admin, see README for creating an "admin" account
+	# you can use separately that will work here:
+	 $(eval OPENSHIFT_TOKEN := $(shell oc whoami -t))
+	docker login -u admin -p $(OPENSHIFT_TOKEN) 172.30.1.1:5000
+	# NOTE: the in-cluster ImageStream tag we use is latest:
+	docker tag cluster-operator:$(MUTABLE_TAG) 172.30.1.1:5000/openshift-cluster-operator/cluster-operator:latest
+	docker push 172.30.1.1:5000/openshift-cluster-operator/cluster-operator:latest
```

Comment:
Particularly wondering about registry IP here.

Reply:
If using cluster up, the registry IP is fixed.
What is the purpose of this target?

Reply:
To bundle all these commands so you've got a simple target to push to the integrate registry, is there a part of this I'm missing?

I could make the IP a variable with this as a default for anyone using minishift or something else.

Reply:
minishift uses cluster up, but yeah, you could make it a variable in case it's done on a different cluster.